functions/v2/http_logging/main.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START functions_structured_logging]
+import functions_framework
+from google.cloud.logging import Client
+
+
+@functions_framework.http
+def structured_logging(request):
+    # Initialize the logging client
```

Comment:
nit: I would probably mention "GCP logging client" in the comment here (and possibly the variable name?). It can get confusing since we're dealing with both the google logging library and the built-in logging library

Reply:
+1, comments should also be reserved to explain _why_ we might be creating a Client and what type it may be, but reiterating what the code is doing in a comment is not absolutely necessary.